src/clients/apiClient.ts

```diff
@@ -9,4 +10,11 @@ export const checkBackendConnection = async (): Promise<boolean> => {
   }
   
   return response.ok;
+
+
+}
+
+export async function fetchSpeciesQuery(tailType: number, size: number, colour: string ) {
+  const response = await fetch(`https://localhost:7127/species?TailType=${tailType}&Size=${size}&Colour=${colour}`);
```

Comment:
We want to use the backend URL in our API call, instead of the local host
```suggestion
  const response = await fetch(`${backendUrl}/species?TailType=${tailType}&Size=${size}&Colour=${colour}`);
```